doc/chsmack.8

```diff
@@ -139,6 +139,13 @@ while still ensuring that the config files of the editor remain protected.
 
 This label is stored in the \fBsecurity.SMACK64TRANSMUTE\fR extended attribute.
 
+.TP
+.B -D, --drop
+
+Use this option to remove the smack properties that are not set for files.
```

Comment:
This description is confusing. "Properties that are not set" can be understood in the chsmack invocation context, as you probably intended, or in the existence of Smack attributes in file system. The latter interpretation raises question - why would I drop a property that is already dropped?

How about an alternative description:
> Use this option to instruct chsmack to remove all attributes that aren't set by other flags. For example chsmack -a Foo -D would drop all attributes except security.SMACK64.